tests/aniso/anisotropic_crash.cc

```diff
@@ -70,7 +70,6 @@ main()
 
           default:
             /// If the number is 3 we don't refine
-            ;
```

Comment:
I guess this one needs to stay.
```
/jenkins/workspace/dealii-serial_PR-18149/tests/aniso/anisotropic_crash.cc:73:9: error: expected primary-expression before ‘}’ token
   73 |         }
      |         ^
```

Reply:
Thank you! Interestingly, my compiler did not complain. I will undo this change!